store/utils.js

```diff
@@ -1,3 +1,79 @@
+import camelcaseKeysDeep from 'camelcase-keys-deep'
+import Swagger from '@aeternity/aepp-sdk/es/utils/swagger'
+import swag from '../swagger.json'
+
+export const initMiddleware = () => {
+  swag.paths = {
+    ...swag.paths,
+    'name/auction/{name}': {
+      get: {
+        operationId: 'getAuctionInfoByName',
+        parameters: [
+          {
+            in: 'path',
+            name: 'name',
+            required: true,
+            type: 'string'
+          }
+        ]
+      }
+    },
+    'txs/backward': {
+      get: {
+        operationId: 'getTxBackward',
+        parameters: [
+          {
+            in: 'query',
+            name: 'type',
+            required: false,
+            type: 'string'
+          },
+          {
+            in: 'query',
+            name: 'type_group',
+            required: false,
+            type: 'string'
+          },
+          {
+            in: 'query',
+            name: 'account',
+            required: false,
+            type: 'string'
+          },
+          {
+            in: 'query',
+            name: 'limit',
+            required: false,
+            type: 'integer'
+          },
+          {
+            in: 'query',
+            name: 'page',
+            required: false,
+            type: 'integer'
+          },
+          {
+            in: 'query',
+            name: 'contract',
+            required: false,
+            type: 'string'
+          }
+        ]
+      }
+    }
+  }
+
+  const swg = Swagger.compose({
+    methods: {
+      urlFor: (path) => process.env.middlewareURL + path,
+      axiosError: () => ''
+    }
+  })({ swag })
+  swg.Swagger = { defaults: {} }
+
+  return camelcaseKeysDeep(swg.api)
```

Comment:
This is fine, but I'm going to fix this on the SDK side https://github.com/aeternity/aepp-sdk-js/issues/1119